### DISCUSSION TITLE
Why this gem over letter_opener?

### DISCUSSION THREAD

**Discussion by MaxLap** [excid3/mailbin#15] [Q&A] [answered]
I'm just wondering if there is a particular reason to use this gem over `letter_opener`, like a killer feature in `mailbin` or a problem with `letter_opener`?

`letter_opener` has existed for basically ever at this point, has 2.6k stars, only 2 issues, 1 open PR, updated last month. I haven't noticed problem with it. So I'm just wondering if I missed something.

This could be a useful information to put in the README.

(I would have made this a discussion, but they are not enabled on this project)

**Comment by excid3** [comment, marked as answer]
Letter opener doesn't have a UI, so you've gotta add a couple gems.

Some different internals here and all in one gem so we can make it Turbo compatible and simpler, modern replacement for letter_opener & letter_opener_web.
